src/hotspot/share/services/mallocHeader.hpp

```diff
@@ -115,15 +122,20 @@ class MallocHeader {
   void set_footer(uint16_t v)       { footer_address()[0] = v >> 8; footer_address()[1] = (uint8_t)v; }
 
  public:
-
+  MallocHeader(const MallocHeader&) = default;
```

Comment:
This makes me a bit nervous, where do we copy malloc headers?

Reply:
Thanks for the catch! It's not necessary, I replaced it with `NONCOPYABLE(MallocHeader)`.